types/webpack/index.d.ts

```diff
@@ -1073,9 +1073,13 @@ declare namespace webpack {
           hooks: {
             jsonpScript?: SyncWaterfallHook<string, Chunk, string>;
             requireExtensions: SyncWaterfallHook<string, Chunk, string>;
+            requireEnsure: SyncWaterfallHook<string, Chunk, string>;
+            localVars: SyncWaterfallHook<string, Chunk, string>;
           };
           outputOptions: Output;
           requireFn: string;
+          renderRequireFunctionForModule(hash: string, chunk: Chunk, varModuleId: number | string): string;
```

Comment:
The param `varModuleId` is marked as optional in the JSDoc annotation: https://github.com/webpack/webpack/blob/v4.41.5/lib/MainTemplate.js#L464

Reply:
Thanks for checking, indeed I have missed it. Changed now to:

```ts
renderRequireFunctionForModule(hash: string, chunk: Chunk, varModuleId?: number | string): string;
```